source/Calamari/Kubernetes/Commands/KubernetesApplyRawYamlCommand.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Calamari.Commands;
+using Calamari.Commands.Support;
+using Calamari.Common.Commands;
+using Calamari.Common.Features.Behaviours;
+using Calamari.Common.Features.ConfigurationTransforms;
+using Calamari.Common.Features.ConfigurationVariables;
+using Calamari.Common.Features.Packages;
+using Calamari.Common.Features.Processes;
+using Calamari.Common.Features.StructuredVariables;
+using Calamari.Common.Features.Substitutions;
+using Calamari.Common.Plumbing.Deployment;
+using Calamari.Common.Plumbing.Deployment.Journal;
+using Calamari.Common.Plumbing.FileSystem;
+using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Deployment;
+using Calamari.Deployment.Conventions;
+using Calamari.FeatureToggles;
+using Calamari.Kubernetes.Conventions;
+using Calamari.Kubernetes.ResourceStatus;
+
+namespace Calamari.Kubernetes.Commands
+{
+    [Command(Name, Description = "Apply Raw Yaml to Kubernetes Cluster")]
+    public class KubernetesApplyRawYamlCommand : Command
```

Comment:
Are you planning to add tests for this in a later PR?

Reply:
Sorry I should have made a comment about testing:
Yes my plan was to 
1. Copy the Kubernetes deployment scripts that currently live in Server to the test project in Calamari so that I could write some extensive tests for all Kubernetes steps. 
2. Once I had completed that, I was going to massage the raw yaml test to instead use this new command so that I could see that it continued to work as expected. 
3. All going well after that I can add new tests to handle the 
4. Once that was done I was going to port the commands in a similar way to this one and I could flick the tests over to ensure I hadn't broken anything.

That whole process is what I was saying could be skipped for now until after the walking skeleton but before the EAP go live.

Reply:
Oh right! Got it! Sounds good to me 👍 
Do you think it'd be worth adding some cards to the board for this for the post walking skeleton phase?

Reply:
I've fleshed out the card that Rob created to include the steps above 👍
https://app.shortcut.com/octopusdeploy/story/47324/move-the-k8s-execution-logic-into-calamari-from-server